contents/handbook/getting-started.md

```diff
@@ -0,0 +1,49 @@
+
+## Big picture
+
+We encourage everyone to start at the beginning first before diving in. We have a strong bias for action, but it is still worth taking a step back and looking at the 'why' first. This helps ensure sure you have the right context and are working on the right things. 
+
+You should start with the '[Company](/handbook/company/story)' section and work your way through everything there. It is not a lot to read. In particular, the sections on our [Strategy](handbook/strategy/strategy) and [Roadmap](/handbook/strategy/roadmap) are a must-read for everyone. 
+
+Next, familiarise yourself with our approach to [Culture](/handbook/company/culture) and our [Values](/handbook/company/values). You might take a bit of time to adjust to PostHog's way of working, and that's ok! In addition to bias for action, you may find that you have a lot more autonomy than you are used to here - you'll realise very quickly that you _shouldn't_ be asking for permission for most things. 
+
+## How we work
+
+Now it's time to dive into some of the more practical stuff - these are the most important pages:
+
+1. [Communication](/handbook/company/communication) - we have a distinctive style. If PostHog is your first all-remote company, this page is especially helpful.
+2. [Team structure](/handbook/people/team-structure/team-structure) - we are structured in Small Teams. These pages will help you get the lay of the land, and who does what. 
+3. [Management](/handbook/company/management) - we have a relatively unusual approach to management, and it is possible that you will not be familiar with our approach. 
+
+### Working in GitHub
+
+We use [GitHub](https://github.com/PostHog) for _everything_, including non-engineering task management. This might take some getting used to if you are non-technical. If that is the case, we have [this intro video](https://www.youtube.com/watch?v=2BB4Nkc0uVM) that will teach you the basics. 
+
+Our most active repositories (aka 'repos') are:
+
+- [PostHog](https://github.com/PostHog/posthog) - main app
+- [PostHog.com](https://github.com/PostHog/posthog.com) - website
+- [Internal](https://github.com/PostHog/internal) - company-facing issues (private)
+
+We use GitHub Projects to track the status of Issues in an easily viewable way. It is up to each Small Team to decide how to manage their tasks, and you'll find most have a dedicated Project - [full list here](https://github.com/orgs/PostHog/projects) - and run two week sprints. As part of onboarding, you will be invited to the relevant planning meetings. 
+
+## Onboarding 
+
+Our [onboarding checklist](https://posthog.com/handbook/people/onboarding) will take you through all the main admin bits you need to get set up, such as accounts access and payroll. The list will vary slightly depending on where you are based and which Small Team you are in. The People team will create an Issue in the Internal repo to track your personal checklist. 
```

Comment:
if you use relative links, the website will be quicker to navigate (ie don't use https://etc)

Reply:
Here this would be `/handbook/people/onboarding`

Reply:
Got it - sorry I was aware of that, but it was a mistake on my part not doing this. Have you made the changes already or is this a comment to say 'hey Charles, make these changes'?

Reply:
This a comment to say 'hey Charles, make these changes' 😂

Reply:
Haha, ok! Done 😄